meta-mender-core/recipes-bsp/u-boot/u-boot-mender-common.inc

```diff
@@ -131,7 +138,7 @@ do_provide_mender_defines() {
 #undef MENDER_BOOTENV_RANGE
 
 #define MENDER_BOOT_KERNEL_TYPE "$MENDER_BOOT_KERNEL_TYPE"
-#define MENDER_KERNEL_NAME "${KERNEL_IMAGETYPE}"
+#define MENDER_KERNEL_NAME "${MENDER_KERNEL_IMAGETYPE}"
```

Comment:
Same, remove curly braces.

Reply:
Are you certain about this one?  The previous line had KERNEL_IMAGETYPE in curly braces.  I have no problem changing it, just want to make sure

Reply:
Yes, because if anyone ever defines a `MENDER_KERNEL_IMAGETYPE` bitbake variable, it will start being expanded as that variable, instead of the shell variable, which is what we want here.